docs/src/developer/future.md

```diff
@@ -0,0 +1,127 @@
+```@meta
+CurrentModule = Nemo
+```
+
+# Future plans
+
```

Comment:
I think this document should clarify what's it's scope is: Is this a roadmap? Things we definitely want to do, or things we might want to do? Is this about short-term or long-term? Is it a mix of all of these? That should be stated here, otherwise I think there will be confusion, and a risk for disappointment when somebody sees something in here they really want and then it never comes, or only comes after 2 years...

Reply:
I leave it to others to decide what this section should be. For now it serves as a list of things that I consider highly  likely to happen based on conversations I've had.